Podfile.lock

```diff
@@ -1,18 +1,18 @@
 PODS:
```

Comment:
I thought we added podfile.lock to the git ignore?

Reply:
I think we discussed it but we didn't actually do it. If we want to we can, but we should also remove Podfile.lock from the repo too.

Reply:
Let's do it. It may be buried in the Storage PR.